### ISSUE TITLE
SKU instead of ID

### ISSUE THREAD

**Issue by KrzysiekWitbee** [duracelltomi/gtm4wp#58]
Hi, I just fund a problem with feature that change ID to SKU. On simple products on event: gtm4wp.addProductToCartEEC information which is pushed to dataLayer contains something like: id: 'SKU: 10001', and it should be id:'10001' without SKU. When do this changes:
Line 718: `"sku"      => $product_sku, `
Deleted: ? __( 'SKU:', 'duracelltomi-google-tag-manager' ) . ' ' . $product_sku : $product_id,
Orginal: "sku"      => $product_sku ? __( 'SKU:', 'duracelltomi-google-tag-manager' ) . ' ' . $product_sku : $product_id,

And now this is working. Maybe you should make this changes in plugin, but I am not sure what is was for ? __( 'SKU:', 'duracelltomi-google-tag-manager' ) . ' ' . $product_sku : $product_id, and what could happen more when I deleted this.

**Comment by duracelltomi** [comment]
Hi,

The "sku" key is not part of the standard enhanced ecommerce product data attributes, I just include it so that everyone can use it if necessary.

Since SKU is not a required attribute of a WooCommerce product, there can be cases when this has not been filled in . The 'SKU: ' prefix has been added to make sure every user can see that the "sku" key has the actual SKU value and not a product ID fallback.

Can you post your use case where you are utilizing this extra "sku" key?

**Comment by KrzysiekWitbee** [comment]
I don't understand your question well. I have written that when I use your option to use SKU instead of ID in Simple Products in ID filed in DataLayer I found "SKU: 1000" instead of just number like "1000", that's why I had to remove this part of the code which I showed in my first post.

And why I need to use SKU instead of ID? Most of my products are Variable Products. ProductDetails are sent to DataLayer only when user choose the variant. I needed something different. I wanted to send info of ProductDetails when user enter the product page no matter if it is a variable or simple product. But in this case, there could be some problems in Google Analytics because there are different IDs for each variant in a variable product. I had to unify this to have one "ID" for every variant that's why I choose to use this option of SKU. Like in this example store: https://enhancedecommerce.appspot.com/ 

Let me know if this explanation is understandable to you :) 
I am also working on more changes needed to have beautiful enhanced ecommerce reports in Google Analytics :)
